py3status/modules/rainbow.py

```diff
@@ -93,11 +98,11 @@ def __init__(self):
     def _init(self):
```

Comment:
Whilst you are here do you want to convert this module to use `post_config_hook()` to do this initialization?

that way we can remove the 
```
        if not self.initialized:
            self._init()
```
check.

We can kill `__init__()`.

Note:   `self.items` should be set properly so that no longer need initializing.